### DISCUSSION TITLE
20-interfaces.conf not installed

### DISCUSSION THREAD

**Discussion by KDGundermann** [pfelk/pfelk#440] [General]
I have just installed pfELK using the script method.
[The wiki](https://github.com/pfelk/pfelk/wiki/References:-Customized-Interface-Names) recommends to change the 20-interfaces.conf file, but this file does not exist on my system, as the install script didn't copy it over in [download_pfelk](https://github.com/pfelk/pfelk/blob/a68b5cf4f952bbf43f9cf9c0269dd9f86a1880e9/etc/pfelk/scripts/pfelk-installer.sh#L1016)

Bug or feature ?

**Comment by a3ilson** [comment]
The script method does not install (download) the 20-interfaces file and it is listed as an optional item during the manual install. It simply enriches the logs but each system will have a varying interface naming schema which is why it was omitted from the base installation. However, downloading and revising as needed is supported.
